app/services/application/ApplicationService.scala

```diff
@@ -114,9 +115,10 @@ trait ApplicationService extends EventSink with CurrentSchemeStatusHelper {
       calculateCurrentSchemeStatus(current, SchemeEvaluationResult(withdrawRequest.schemeId, EvaluationResults.Withdrawn.toString) :: Nil)
     }
 
-    def maybeProgressToFSAC(schemeStatus: Seq[SchemeEvaluationResult]) = {
+    def maybeProgressToFSAC(schemeStatus: Seq[SchemeEvaluationResult], latestProgressStatus: Option[ProgressStatus]) = {
       val greenSchemes = schemeStatus.collect { case s if s.result == Green.toString => s.schemeId }.toSet
-      val shouldProgressToFSAC = greenSchemes subsetOf schemesRepo.nonSiftableSchemeIds.toSet
+      val shouldProgressToFSAC = !latestProgressStatus.contains(ProgressStatuses.SIFT_READY) &&
```

Comment:
Did you test this with a few scenarios? I don't think this will work if you have `[HR, Generalist, DAT]` and then withdraw from DAT before completing the forms. This check will stop the candidate from progressing even though they don't need to complete any forms for the remaining schemes, i.e `[HR, DAT]`